config/config.yaml

```diff
@@ -50,23 +50,43 @@ solver: "cbc" # cbc, cplex, gurobi
 
 user_defined_capacity:
   # technology: [capacity, 
-  #              first_year, 
+  #              build_year, 
   #              first_year_of_expansion, 
   #              build_rate_per_year, 
-  #              cost, 
+  #              capex_cost, 
   #              efficiency]
   PWRCOAINDWE01: [8, 2000, 2025, 5, 1100, 35]
   PWRBIOINDWE01: [0, 2020, 2030, 2, 2000, 28]
 
 user_defined_capacity_transmission:
-  # technology: [capacity, 
-  #              first_year, 
-  #              first_year_of_expansion, 
-  #              build_rate_per_year, 
-  #              cost, 
-  #              efficiency]
-  TRNINDEAINDNE: [5, 1975, 2030, 10, 861, 95]
-  TRNINDNOINDSO: [0, 2020, 2025, 2, 1800, 92]
+  # entry: [technology,
+  #         capacity, 
+  #         build_year, 
+  #         first_year_of_expansion, 
+  #         build_rate_per_year, 
+  #         capex_cost,
+  #         annual_fixed_O&M_cost,
+  #         efficiency]
+  trn1: [TRNINDEAINDNE, 5, 1975, 2030, 10, 861, 30, 95]
+  trn2: [TRNINDEAINDNE, 1, 1990, 2030, 10, 861, 30, 95]
+  trn3: [TRNINDEAINDNE, 2, 2035, 2030, 10, 861, 30, 95]
+  trn4: [TRNINDNOINDSO, 0, 2020, 2025, 0.5, 900, 32, 92]
+  # Per technology, only the capacity and build_year
+  # parameters can deviate from each other. For all other
+  # parameters only the last entry will be used.

```

Comment:
Why the change to defining `trn1`, `trn2`, ect... ?

Reply:
The entries are read into the different scripts as dictionaries. Keeping the `technology` name as key would lead to the different residual capacity entries to be read incorrectly. @trevorb1 If you have a better alternative please do let me know (I didn't love setting it up this way either but ran out of brain space... ).

Reply:
Ahhh fair enough; good point! I think until we come up with a better method to do user defined capacity, this works just great. :)